src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -38,19 +41,28 @@ const ProfileHeader = ({
   );
 };
 
-const mapStateToProps = state => ({
-  // moving to state.user.hero.userFullName
-  userFullName: state.user.profile.userFullName,
-});
+const mapStateToProps = state => {
+  const latestBranchOfService = orderBy(
+    state?.vaProfile?.militaryInformation?.serviceHistory?.serviceHistory,
+    ['endDate'],
+    ['desc'],
```

Comment:
I think the above 2 lines can just be strings instead of an array with 1 string, Lodash should be able handle these args as strings 🙂

Reply:
@kelsonic when I put the iteratees as a string, it throws an error asking for an array, but the last arg can accept a string.

Reply:
Oh interesting! Sounds good!